runtime/src/lib.rs

```diff
@@ -189,16 +189,12 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
 
 pub fn handle_mailbox_commands(drivers: &mut Drivers) {
     // Indicator to SOC that RT firmware is ready
-    drivers
-        .soc_ifc
-        .regs_mut()
-        .cptra_flow_status()
-        .write(|w| w.ready_for_runtime(true));
+    drivers.soc_ifc.assert_ready_for_runtime();
     caliptra_drivers::report_boot_status(RtBootStatus::RtReadyForCommands.into());
     loop {
         wait_for_cmd(&mut drivers.mbox);
-
         if drivers.mbox.is_cmd_ready() {
+            caliptra_common::wdt::start_wdt(&mut drivers.soc_ifc, WdtTimeout::default());
```

Comment:
Could put the start/stop in `wait_for_cmd`. This more closely aligns with John's request that the WDT is handled directly before/after wfi.

Reply:
we don't have wfi yet - we would keep hammering the register with no wait.

Reply:
Ack. Can you add a TODO comment here?

Reply:
Sure. Done.